src/Infrastructure/Form/Regulation/VehicleSetFormType.php

```diff
@@ -42,12 +41,20 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             )
             ->add(
                 'heavyweightMaxWeight',
-                NumberType::class,
-                options: [
+                ChoiceType::class,
+                options : [
+                    'choices' => [
+                        'regulation.vehicle_set.heavyweightMaxWeight.3.5t' => 3.5,
+                        'regulation.vehicle_set.heavyweightMaxWeight.7.5t' => 7.5,
+                        'regulation.vehicle_set.heavyweightMaxWeight.19t' => 19,
+                        'regulation.vehicle_set.heavyweightMaxWeight.26t' => 26,
+                        'regulation.vehicle_set.heavyweightMaxWeight.32t' => 32,
+                        'regulation.vehicle_set.heavyweightMaxWeight.44t' => 44,
+                    ],
```

Comment:
Si j'ai bien compris les traductions corresopndent exactement au float du tonnage, donc je pense que tu peux simplifier en utilisant un simple array ?

```suggestion
                    'choices' => [
                        3.5,
                        7.5,
                        19,
                        26,
                        32,
                        44,
                    ],
```

et retirer les clés de traductions

Reply:
Alors je viens de faire la modif et ça m'affiche 
![Capture d’écran du 2024-12-18 14-51-06](https://github.com/user-attachments/assets/0bff4599-66ea-4903-afcc-147e3ff2d932)

Reply:
@florimondmanca

Reply:
OK alors il faut sûrement faire un array avec clé et valeur identique, je pense qu'il utilise l'index dans l'array comme label d'où 0 1 2 etc...

```php
'choices' => [
    3.5 => 3.5,
    // etc
]
```

Un peu fastidieux mais bon